src/services/getBatteryIcon.js

```diff
@@ -29,7 +28,7 @@ const getBatteryIcon = (batteryType, batteryPercentage) => {
     if (batteryPercentage < 90 && batteryPercentage > 30) {
       return <LockBatteryHalfSvg />;
     }
-    if (batteryPercentage < 30) {
+    if (batteryPercentage <= 30) {
```

Comment:
You need this `less than or equal` comparison on `batteryType === 'main'` condition too